api/preprints/serializers.py

```diff
@@ -147,3 +154,88 @@ def set_node_field(self, func, val, auth):
             raise exceptions.PermissionDenied('Not authorized to update this node.')
         except ValueError as e:
             raise exceptions.ValidationError(detail=e.message)
+
+
+class PreprintDetailSerializer(PreprintSerializer):
```

Comment:
This class and `PreprintDetailRetrieveSerializer` are now unused, please remove them